providers/openfeature-provider-flagd/pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
   "panzi-json-logic>=1.0.1",
   "semver>=3,<4",
   "pyyaml>=6.0.1",
+  "cachebox"
```

Comment:
Are there any concerns about this dependency? It seems to be Rust-based. Will that be an issue for consumers?

Reply:
background from myside, why i went with cachebox: